docs/source/examples.rst

```diff
@@ -415,3 +415,71 @@ ISO code: its 3-digit ISO code and an internal class name.
     True
 
 .. deprecated:: In the future
+
+Generate iCalendar content and export to ``.ics``
+-------------------------------------------------
+
+:py:func:`ICalExporter` facilitates the creation and export of iCalendar files in compliance with `RFC 5545 <https://datatracker.ietf.org/doc/html/rfc5545>`__.
+
+The iCalendar specification only supports 2-letter `ISO 639-1 <https://www.loc.gov/standards/iso639-2/php/code_list.php>`__ language codes, which means some holiday lists may not be exportable.
+For instance, Aruba's ``pap_AW`` localization is unsupported. Similarly, all of Hong Kong's Chinese language localizations (``zh_CN``, ``zh_HK``) are assigned the shared ``ZH`` code.
+
+Furthermore, if no :py:attr:`language` is specified for a given holiday but :py:attr:`default_language` is set for that Holiday object, the default language will be used instead.
+If neither attribute exists, the fallback language code ``EN`` will be applied.
+
+.. code-block:: python
+
+   >>> from holidays import country_holidays
+   >>> from holidays.ical import ICalExporter
+   >>> us_holidays = country_holidays('US', years=2020)
+   # Initializes iCalendar exporter.
+   >>> exporter = ICalExporter(us_holidays)
+   # Initializes iCalendar exporter with custom ``DTSTAMP`` for all ``VEVENT``,
```

Comment:
Formatting with \`\`...\`\` is not supported in the documentation in embedded code comments.